Docs: move the cookbook to its own how-to page (#4487)
aiidateam/aiida-core: docs/source/howto/cookbook.rst

        transport = computer.get_transport()
        scheduler = computer.get_scheduler()
        scheduler.set_transport(transport)

        # This opens the SSH connection, for SSH transports
        with transport:
            if only_current_user:
                remote_username = transport.whoami()
                all_jobs = scheduler.get_jobs(user=remote_username, as_dict=True)
            else:
                all_jobs = scheduler.get_jobs(as_dict=True)

        return all_jobs

    if __name__ == '__main__':
        all_jobs = get_scheduler_jobs(only_current_user=False)
        user_jobs = get_scheduler_jobs(only_current_user=True)

        print(f'Current user has {len(user_jobs)} jobs out of {len(all_jobs)} in the scheduler'
        print('Detailed job view:')

        for job_id, job_info in user_jobs.items():
            print(f'Job ID: {job_id}')
            for k, v in job_info.items():
                if k == 'raw_data':
                    continue
                print(f'  {k}: {v}')
            print('')

Use ``verdi run`` to execute it:

.. code-block:: console

    verdi run file_with_script.py

.. important::

>     Every time you call the function, an SSH connection is executed!

Reviewer: ```suggestion
    Every time you call the function, two SSH connections are executed!
```
